app.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const app = express();
+const {getCategories} = require('./controllers/controller');
+
+
+app.get("/api/categories", getCategories);
+
+
+app.all('/*', (req, res) => {
```

Comment:
Might be nice to seperate this out to an error file to keep the app.js file as readable as possible, It might be nice to handle server errors (500's) at this stage aswell incase of database outages